README.md

```diff
@@ -169,6 +169,8 @@ for an example workspace using another scala version.
 | 0.14.x | 3b9ab9be31ac217d3337c709cb6bfeb89c8dcbb1 |
 | 0.13.x | 3c987b6ae8a453886759b132f1572c0efca2eca2 |
 
+PRs are also built with latest released bazel version (see [travis config](https://github.com/bazelbuild/rules_scala/blob/master/.travis.yml#L66))
```

Comment:
Wdyt about changing this to say with highest supported bazel version (right now it's latest but there were times where this wasn't true).
We can say (see Travis config for the exact highest version)